main.tf

```diff
@@ -85,3 +85,48 @@ resource "ibm_database" "mongodb" {
     ]
   }
 }
+
+##############################################################################
+# Context Based Restrictions
+##############################################################################
+
+module "cbr_rule" {
+  count            = length(var.cbr_rules) > 0 ? length(var.cbr_rules) : 0
+  source           = "git::https://github.com/terraform-ibm-modules/terraform-ibm-cbr//cbr-rule-module?ref=v1.1.2"
+  rule_description = var.cbr_rules[count.index].description
+  enforcement_mode = var.cbr_rules[count.index].enforcement_mode
```

Comment:
this reference is incorrect, as it is currently implemented enforcement mode is not in the cbr_rules variable. Should be just `var.enforcement_mode`